core/scripts/chaincli/handler/debug.go

```diff
@@ -27,7 +29,6 @@ import (
 	"github.com/smartcontractkit/chainlink/v2/core/services/ocr2/plugins/ocr2keeper/evm21/encoding"
 	"github.com/smartcontractkit/chainlink/v2/core/utils"
 	bigmath "github.com/smartcontractkit/chainlink/v2/core/utils/big_math"
-	ocr2keepers "github.com/smartcontractkit/ocr2keepers/pkg/v3/types"
```

Comment:
i think this import needs to stay here.

Reply:
interestingly this is auto moved to the above line whenever i save the file :)